### PR TITLE
isotowbfs.com.ISOtoWBFS version 1.0.0.0

### PR DESCRIPTION
**Pull request by techflashYT**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/47426)

### CODE REVIEW

**Comment by techflashYT** [comment]
Hmm, maybe the PHP redirecting link will work, it seems like it allocates an id.... one second while I test that out.

**Comment by techflashYT** [comment]
hmm, I just checked it, it installs with `winget install -m manifests/i/isotowbfs/com/ISOtoWBFS/1.0.0.0`,  I'm not sure why it doesn't work here.  I also tried completely remaking the manifest from scratch with the new URL, but it still didn't work, and git isn't reporting any changes from the previous commit.

File: manifests/i/isotowbfs/com/ISOtoWBFS/1.0.0.0/isotowbfs.com.ISOtoWBFS.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Created using wingetcreate 0.5.0.1
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: isotowbfs.com.ISOtoWBFS
+PackageVersion: 1.0.0.0
+PackageLocale: en-US
+Publisher: isotowbfs.com
+PackageName: ISO to WBFS
+License: N/A
```
Comment (ItzLevvie) [inline manifests/i/isotowbfs/com/ISOtoWBFS/1.0.0.0/isotowbfs.com.ISOtoWBFS.locale.en-US.yaml:9]:
```suggestion
License: Freeware
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by Trenly** [comment]
The PHP link won't work here since it gives an HTML page, which is why the Hash mismatch is occuring.
The direct file link won't work either since the PHP creates a unique download link for each user. 

To add this package to the repository, the publisher will need to provide a static link to download the file

**Pull request closed without merging**